tests/test_ratmama.py

```diff
@@ -0,0 +1,244 @@
+"""
+test_ratmama.py - Tests for the Ratmama.py
+
+Tests the parsing facilities for ratsignals
+
+Copyright (c) 2018 The Fuel Rat Mischief,
+All rights reserved.
+
+Licensed under the BSD 3-Clause License.
+
+See LICENSE.md
+"""
+
+import pytest
+from Modules.rat_board import RatBoard
+from tests.mock_callables import AsyncCallableMock
+from Modules.context import Context
+import Modules.RatMama as RatMama
+from Modules.rat_rescue import Platforms
+from Modules.rat_rescue import Rescue
+
+pytestmark = [pytest.mark.ratsignal_parse, pytest.mark.asyncio]
+
+
+class TestRSignal(object):
+    rat_board: RatBoard
+
+    async def test_ratmama_wrong_platform(self, context_channel_fx: Context,
+                                                 monkeypatch
+                                          ):
+        """
+        Tests, that the parser does not implode upon being given a wrong platform by RatMama
+        """
+        monkeypatch.setattr(context_channel_fx, '_words_eol',
+                            ["Incoming Client: Ajdacho - System: Alrai - "
+                             "Platform: Switch - O2: OK - Language: Polish (pl-PL)"
+                             ]
+                            )
+
+        # now we just set the nickname to the allowed on
+        monkeypatch.setattr(context_channel_fx._user, '_nickname', "RatMama[BOT]")
```

Comment:
Another hard-coded nickname.  Let's discuss and see if there's a better way, like pulling the config value for the set nickname for continuity.